lib/config.js

```diff
@@ -88,6 +88,11 @@ module.exports = class Config {
       res.server.middleware.unshift(hygienist(res.server.server.baseDir))
     }
 
+    // webpack must consume an array for the value in our entry object
+    for (let key in res.entry) {
+      if (typeof value === 'string') res.entry[key] = Array.prototype.concat(res.entry[key])
+    }
```

Comment:
Ah sorry you can actually leave out the type check as well, this will leave an array as an array but coerce a string. Also could you add a test for this? 👼 🙇